### ISSUE TITLE
val data is used for training?

### ISSUE THREAD

**Issue by turboxin** [skyhehe123/SA-SSD#36]
In https://github.com/skyhehe123/SA-SSD/blob/9bb2ef4aecc7206ea935977d45781267b8a15001/tools/create_data.py#L278 I noticed that groundtruth in val set is also created into groundtruth_database, which means val data is used for training?

**Comment by skyhehe123** [comment]
@turboxin The train/val split is a human defined protocol for ablation study in many papers. When evaluation on the test set (for KITTI server comparison), it is free to use the whole/proportion of training data.
